<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MessageItem</class>
 <widget class="QWidget" name="MessageItem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>260</width>
    <height>70</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="styleSheet">
      <string notr="true">QGroupBox {background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,stop: 0 #40E0E0, stop: 1 #FFFFFF);border: 2px solid gray;margin-top: 0px; }</string>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QLabel" name="label_name">
      <property name="geometry">
       <rect>
        <x>4</x>
        <y>45</y>
        <width>61</width>
        <height>24</height>
       </rect>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel{
	border-color: rgb(23, 30, 5);
    font: 100 12pt &quot;微软雅黑&quot;;
    color: rgb(33, 250, 127);
    background-color: rgba(255, 193, 245, 0%);
}
QLabel:hover{
	border-color: rgb(0, 170, 255);
 
	color: rgb(255, 0, 0);
	font: 100 20pt &quot;微软雅黑&quot;;
    background-color: rgba(255, 193, 245, 0%);
}</string>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;瓜不甜&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_Photo">
      <property name="geometry">
       <rect>
        <x>9</x>
        <y>4</y>
        <width>40</width>
        <height>40</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>40</width>
        <height>40</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>40</width>
        <height>40</height>
       </size>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel{
	border-color: rgb(23, 30, 5);
  /*  border: 1px groove gray;*/
    background-color: rgba(255, 193, 245, 0%);
}
QLabel:hover{
	border-color: rgb(0, 170, 255);
    border: 4px groove gray;
    background-color: rgba(255, 193, 245, 0%);
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="resource.qrc">:/resource/image/login/resource/image/login/avatar.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="btReaded">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>9</y>
        <width>61</width>
        <height>51</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	border: 1px solid #c9c9c9;
	background: #569082;
	color:#ffffff;
	font-family: &quot;微软雅黑&quot;;
	font-size: 15px;
}
QPushButton:hover {
	border: 1px solid #c9c9c9;
	background: #ff9032;
	color:#ffffff;
	font-family: &quot;微软雅黑&quot;;
	font-size: 15px;
   
}
QPushButton:pressed {
	border:9px solid #c9c9c9;
	background: #569082;
	color:#ffffff;
	font-family: &quot;微软雅黑&quot;;
	font-size: 20px;
   
}</string>
      </property>
      <property name="text">
       <string>标读</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_message">
      <property name="geometry">
       <rect>
        <x>66</x>
        <y>27</y>
        <width>121</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>我去恶趣味金额sssadjaskjdkasjdklasdk阿斯达斯</string>
      </property>
      <property name="textFormat">
       <enum>Qt::AutoText</enum>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_time">
      <property name="geometry">
       <rect>
        <x>63</x>
        <y>5</y>
        <width>121</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">/*全局设置*/
QLabel {	font-family: &quot;微软雅黑&quot;; font-size: 12px;
			


}</string>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; font-weight:600; color:#b0021c;&quot;&gt;3月4日 下午3：45&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
